### ISSUE TITLE
Best practice: test build without actually deploying

### ISSUE THREAD

**Issue by dfberry** [Azure/static-web-apps#674]
How can I test the action completely without actually deploying it - Is there a property I can set or a different action I should use?

**Comment by rhalaly** [comment]
Azure Static Web Apps is a wrapper of Web App + Azure Functions API. You can test each one separately as you would do without Azure Static Web Apps, based on the libraries/technology you chose.

**Comment by miwebst** [comment]
That isn't quite right. While Azure Static Web Apps supports a lot of the same behaviors as using a Web App + Azure Function in conjunction, there is a lot more (i.e. global distribution, free SSL certs) that is added as well. It isn't simply a wrapper.

@dfberry you could have a separate workflow that runs on every commit to simply run the build / test components without deploying. To do this, you can remove the ' azure_static_web_apps_api_token' from your workflow file inputs and add 'skip_deploy_on_missing_secrets' as an environment variable with the value 'true'. Similar to this thread: https://github.com/Azure/static-web-apps/issues/679

**Comment by rhalaly** [comment]
@miwebst of course. I meant that the code is separated from Azure Static Web Apps hosting and the code can be built as a regular app.
 
Azure Static Web Apps is super cool service :)
